src/components/Communicator/Communicator.actions.js

```diff
@@ -230,12 +234,15 @@ export function verifyAndUpsertCommunicator(
  */
 
 export function getApiMyCommunicators() {
-  return dispatch => {
+  return async dispatch => {
     dispatch(getApiMyCommunicatorsStarted());
     return API.getCommunicators()
       .then(res => {
         dispatch(getApiMyCommunicatorsSuccess(res));
-        return res;
+
+        return dispatch(syncCommunicators(res.data)).catch(e => {
+          console.error(e);
+        });
```

Comment:
would be better to return the `res` after the dispatch of `syncCommunicators`. to be more consistence with the old behavior.